crypto3/libs/zk/include/nil/crypto3/zk/snark/arithmetization/plonk/constraint_system.hpp

```diff
@@ -50,8 +50,10 @@ namespace nil {
     namespace crypto3 {
         namespace zk {
             namespace snark {
-
                 /************************* PLONK constraint system ****************************/
+                constexpr static int const PLONK_SPECIAL_SELECTOR_ALL_USABLE_ROWS_SELECTED = -1;
```

Comment:
Let's try to use std::numeric_limits<size_t>::max - 1, I'm afraid of negative values, they may get converted to unsigend somewhere by a mistake.

Reply:
Done in last commit